.github/workflows/backend-tests.yml

```
@@ -0,0 +1,87 @@
+      DB_HOST: 127.0.0.1
+      DB_PORT: "5432"
+      DB_NAME: remotecheckin_test
+      FLASK_ENV: testing
+      JWT_SECRET_KEY: test-secret-key-with-numbers-123
+      ENCRYPTION_KEY: KlRqjEOThj9Yket0vV7M9AQLXmBu5ssxg_JudOYiakI=
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.10)

[error] the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)